book/src/configuration.md

```diff
@@ -332,3 +332,23 @@ max-wrap = 25 # increase value to reduce forced mid-word wrapping
 max-indent-retain = 0
 wrap-indicator = ""  # set wrap-indicator to "" to hide it
 ```
+
+### `[editor.sticky-context]` Section
+
+Option for sticky context, which is a feature that sticks bigger blocks of code
+e.g. Functions to the top of the viewport
+
+| Key         | Description                                                                                | Default |
+| ---         | ---                                                                                        | ---     |
+| `enable`    | Display context of current line if outside the view                                        | `false` |
+| `indicator` | Display an additional node to indicate what part of the view is the sticky context         | `false` |
```

Comment:
Would it be better to put a different background color instead of having a sticky line there? That way we can save one line from being unused.

Reply:
> Ah, I just realized the indicator is optional. Should this be called "line" instead of "node"?

i have no strong feelings about the name.
I would argue that 'node' is more correct, but if it suffers from understand-ability, i am happy to change it :)